cli/cache.rs

```diff
@@ -84,51 +145,22 @@ impl FetchCacher {
       root_permissions,
     }
   }
-
-  fn get_emit_metadata(
-    &self,
-    specifier: &ModuleSpecifier,
-  ) -> Option<EmitMetadata> {
-    let filename = self
-      .disk_cache
-      .get_cache_filename_with_extension(specifier, "meta")?;
-    let bytes = self.disk_cache.get(&filename).ok()?;
-    serde_json::from_slice(&bytes).ok()
-  }
-
-  fn set_emit_metadata(
-    &self,
-    specifier: &ModuleSpecifier,
-    data: EmitMetadata,
-  ) -> Result<(), AnyError> {
-    let filename = self
-      .disk_cache
-      .get_cache_filename_with_extension(specifier, "meta")
-      .unwrap();
-    let bytes = serde_json::to_vec(&data)?;
-    self.disk_cache.set(&filename, &bytes).map_err(|e| e.into())
-  }
 }
 
 impl Loader for FetchCacher {
   fn get_cache_info(&self, specifier: &ModuleSpecifier) -> Option<CacheInfo> {
     let local = self.file_fetcher.get_local_path(specifier)?;
     if local.is_file() {
       let location = &self.disk_cache.location;
-      let emit = self
-        .disk_cache
-        .get_cache_filename_with_extension(specifier, "js")
-        .map(|p| location.join(p))
-        .filter(|p| p.is_file());
-      let map = self
-        .disk_cache
-        .get_cache_filename_with_extension(specifier, "js.map")
-        .map(|p| location.join(p))
-        .filter(|p| p.is_file());
+      let emit =
+        get_cache_filename_with_extension(specifier, EMIT_CACHE_DATA_EXT)
+          .map(|p| location.join(p))
+          .filter(|p| p.is_file()); // todo: dsherret: this condition is racy so we should remove it
       Some(CacheInfo {
         local: Some(local),
-        emit,
-        map,
+        // todo(dsherret): deno_graph should not have multiple files for emit and map
+        emit: emit.clone(),
+        map: emit,
       })
```

Comment:
What is this `CacheInfo` used for? I forget and didn't figure it out on a quick glance as it doesn't seem to be used (unless my editor is failing me).